### DISCUSSION TITLE
[BUG/Help] How to evaluate ChatGLM

### DISCUSSION THREAD

**Discussion by liuxiaocs7** [zai-org/ChatGLM-6B#1375] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

How to evaluate ChatGLM using scripts from https://github.com/THUDM/GLM-130B

### Steps To Reproduce

No.

### Environment

```markdown
- OS:
- Python:
- Transformers:
- PyTorch:
- CUDA Support (`python -c "import torch; print(torch.cuda.is_available())"`) :
```

**Comment by Hap-Zhang** [comment]
+1
